epix/io.py

```diff
@@ -227,15 +227,18 @@ def awkward_to_wfsim_row_style(interactions):
         res['g4id'][i::2] = awkward_to_flat_numpy(interactions['evtid'])
         res['vol_id'][i::2] = awkward_to_flat_numpy(interactions['vol_id'])
         res['e_dep'][i::2] = awkward_to_flat_numpy(interactions['ed'])
-        
+        if 'local_field' in res.dtype.names:
```

Comment:
Why do we need the if case here? `res.dtype` is taken from WFsim defined [here](https://github.com/XENONnT/WFSim/blob/master/wfsim/strax_interface.py#L31-L32) I think so the condition is always true?

Reply:
It is backwards compatibility, when epix is used with older version of WFsim. Probably not really necessary, but I prefer to keep compatibility for +- few versions of the code.